wicket/src/ui/panes/overview.rs

```diff
@@ -439,3 +444,429 @@ impl Control for InventoryView {
         }
     }
 }
+
+fn inventory_description(component: &Component) -> Text {
+    let sp = component.sp();
+
+    let label_style = Style::default().fg(colors::OX_OFF_WHITE);
+    let ok_style = Style::default().fg(colors::OX_GREEN_LIGHT);
+    let bad_style = Style::default().fg(colors::OX_RED);
+    let warn_style = Style::default().fg(colors::OX_YELLOW);
```

Comment:
Could these be moved to `style.rs`? To address #3568 we'll likely have to switch to a stylesheet approach, and moving this to `style.rs` would be a nice step towards that.

Reply:
Actually I think these are already exposed in `style.rs`, so maybe just use those?

Reply:
Yep, moved in 88883027